rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridgeManager.java

```diff
@@ -45,13 +45,9 @@
 
     private final Map<String, Xid> transactionXids;
 
-    public static InboundBridgeManager getInstance() {
+    public static synchronized InboundBridgeManager getInstance() {
```

Comment:
Yes, I was proposing:
            synchronized (InboundBridgeManager.class) {		+            INSTANCE = new InboundBridgeManager();
                if (INSTANCE == null) {		
                    INSTANCE = new InboundBridgeManager();		
                }			 
                return INSTANCE;	
            }

Which is basically what you have anyway